common/include/common/common_utils.hpp

```diff
@@ -736,9 +742,19 @@ static inline double time_event(event_t&);
 template <typename event_t>
 static inline double time_events(std::vector<event_t> es) {
   double total_time = 0;
+#if defined(BUILD_CUBLAS_BENCHMARKS)
```

Comment:
Remove this timing here and create a cuda specific `time_events` function inside `cublas/utils.hpp`